modules/uhsg_feedback/uhsg_feedback.module

```diff
@@ -58,7 +58,7 @@ function uhsg_feedback_mail_alter(&$message) {
 
     // I would like to get a response to my feedback.
     if ($contact_message->get('field_feedback_respond')->value) {
-      $message['body'][] = t('I would like to get a response to my feedback: @replyTo', ['@replyto' => $replyTo]);
+      $message['body'][] = t('I would like to get a response to my feedback: @replyTo', ['@replyTo' => $replyTo]);
```

Comment:
Had to doublecheck this change. 

Seems like the old version never had this @replyTo-replacement working due to a missing uppercase. Good that you didnt miss this :)

Reply:
Yep, noticed it when testing the feedback emails. A lucky find :)